Specs/Scene/MetadataTableSpec.js

```diff
@@ -64,7 +64,7 @@ describe("Scene/MetadataTable", function () {
     );
   });
 
-  it("constructor throws without count", function () {
+  it("constructor throws without parameters", function () {
```

Comment:
I think we do want to keep this as "throws without count", just pass in `{}` for class